ogre_mesh_viewer.py

```diff
@@ -573,11 +577,14 @@ def initScene(self):
             self.camman = OgreBites.CameraMan(camnode)
         else:
             self.camman.setCamera(camnode)
+            self.camman.setTarget(camnode.getCreator().getRootSceneNode())
```

Comment:
I guess this is the relevant line from this commit and the rest is debug leftover?

Reply:
Yes, but I would like to leave the debug code.

Perhaps, it should be marked as DEBUG log.

What do you think?